docs/simple-application.md

```diff
@@ -4,7 +4,15 @@ Let's develop a simple application that covers some of the major aspects of Sing
 
-First let's create an entry point for the application. Create a file `index.html`:
+---
+
+We could create the entire application in a single Javascript file, but doing so would make it difficult to navigate the codebase later on. Instead, let's split the code into *modules*, and assemble these modules into a *bundle* `bin/app.js`.
+
+There are many ways to setup a bundler tool, but most are distributed via NPM. In fact, most modern Javascript libraries and tools are distributed that way, including Mithril. NPM stands for Node.js Package Manager. To download NPM, [install Node.js](https://nodejs.org/en/); NPM is installed automatically with it. Once you have Node.js and NPM installed, install Mithril and create the project skeleton.
+
+To install Mithril, follow the instructions in the [installation](installation.md) page. Once you have completed the seven steps outlined in [Quick Start with Webpack](https://mithril.js.org/installation.html#quick-start-with-webpack), we are ready to build upon the project skeleton that you have created.
```

Comment:
Could you change the link to `[[Quick Start with Parcel]](...#quick-start-with-parcel)`?

Reply:
@pygy Is there such a link? I'm not seeing that anywhere. I just see the webpack heading and instructions.

Reply:
@WildYorkies it is in the `next` branch, it hasn't been published yet